### ISSUE TITLE
GeoLootCrate

### ISSUE THREAD

**Issue by MCDForm** [district0x/district-proposals#239]
## **Name**: 
GeoLootCrate

## **Purpose**: 
Geocaching site with digital loot containing caches.

## **Description**: 
Geocaching is an outdoor recreational activity, in which people use a Global Positioning System (GPS) receiver or mobile device and other navigational techniques to hide and seek containers, called "geocaches" or "caches", at specific locations marked by coordinates all over the world.  

In the past the geocaches have been physical containers with small trinkets or notepads.  This district will change that using a system of advertiser driven digital loot crate style caches.

This district allows  businesses, charities, schools (any entity) to create a geocache that contains data with stored value.  This data can be a coupon, credit, special promotion, access code, digital collectible or any other type of digital asset. 

The end user who is a Geocacher will use a web/mobile interface to find geocaches near their location.  When they reach the geolocation they engage the smart contract to execute.  The smart contract will then any of the previously mentioned items to the user.

The marketplace aspect of this district is rear facing and meant for the advertisers/entities rather than the end user.  These customers will purchase smart contracts and load their own GPS coordinates, award data, graphics and details such as number of rewards available or expiration date.

When the marketplace smart contract is purchased and activated it will display in an easy to use User Interface for consumers.  The User Interface will appear similar to popular geocaching sites like Geocaching.com.

All transactions will be currency agnostic but will initially use only Ether.  All gas prices will be rolled into the smart contract and paid for by the original purchaser of the smart contract using Bokky’s Token Teleportation Service (BTTS).

Businesses will also be able to purchase event smart contracts that can be for private distribution.  See use cases for an example.

### Extra Bonuses
1.  Geocaching with physical containers is not allowed everywhere like National Parks.
1.  The blockchain allows for the transfer of digital assets (cryptocurrency, ERC271 tokens, etc) that contain value.
1,  Due to its digital nature it is possible to also create geocaches in virtual worlds like Decentraland.

### Use Cases
1. Hiking - Bob is geocaching with his family in Joshua Tree and locates a cache near Jumbo Rocks.  The cache provides Bob with a user specific QR code for REI that gives him 25% off rock climbing equipment.
1. Upper Deck baseball cards would like to provide a limited number of ERC721 cryptocollectible baseball cards but would like to target baseball fans.  Fans access the cache by scanning a QR code provided on their ticket stub.

## **Ethereum Address**:
0x1c63a09C62558B2A614f48e7495291f02f99AC40

**Comment by Bradymck** [comment]
Now that mobile dApps can be used in browsers like Cipher Browser, Toshi, Status and Tokenary I wonder how hard it would be to utilize the GPS to make this happen within a dApp. 

Or would you take a centralized approach? I'm honestly leaning towards hybrid centralized/decentralized apps to make a better user experience but still have a full decentralized mode that you can opt in for.

**Comment by MCDForm** [comment]
I'm thinking that for many game style apps the hybrid approach will work better.  I was spending time in a National Park last week and noticed that there's limits to geocaching due to the protected nature of the land.  It would also be interesting to integrate into virtual worlds somehow.

**Comment by Bradymck** [comment]
I agree, a VR approach may be interesting to bootstrap the concept before launching IRL. High Fidelity is open source and has a cryptocurrency called HFC. May be interesting to get it going in there.

**Comment by Bradymck** [comment]
Soon will be able to use Decentraland as well. Though I think a cross platform VR app would be huge. Make it as open as possible regardless of what people use.

**Comment by popcorngeneral** [comment]
Like Lomocoin (LMC)?

**Comment by Bradymck** [comment]
Hey @MCDForm 

I'm doing something similar to the NFT based loot boxes on stream now. Check it out if you're still around: https://dappstream.live/cloudbot

These could be used like you describe utilizing Linkdrop like I do.
